common/src/commonMain/sqldelight/com/cuhacking/atlas/db/Database.sq

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE features (
+  fid TEXT NOT NULL,
```

Comment:
```suggestion
  fid INTEGER PRIMARY KEY NOT NULL,
```

Reply:
Oops, I didn't pay attention to the type! Would it be a good idea to make the id autoincrement and change the insert query so that you don't have to enter an id?

Reply:
The IDs will be supplied along with the rest of the data, so we won't need to autoincrement anything in this case.